### DISCUSSION TITLE
Iframe inside APP on iOS not working

### DISCUSSION THREAD

**Discussion by syshex** [ionic-team/capacitor#3459] [Q&A]
Hey everyone,

Developing an app with Quasar , using capacitor. 
Inside this app, I have an Iframe.   

I build the app for both android and iphone, and on the android, it works fine, as expected.  On the iphone, the iframe does not load the content.  

Has anyone been through this problem and can lend me a hand?

Just for completeness, the iframe code at this moment, for testing is : 

```
        <iframe src="https://www.openstreetmap.org" style="width: 100%; height: 100%"></iframe>
```

 I've tried adding to capacitor.config.json : 
```
{
  "appId": "com.aaa.bbb",
  "appName": "The test Project",
  "bundledWebRuntime": false,
  "npmClient": "yarn",
  "webDir": "www",
  "server": {
      "allowNavigation": [
          "*",
          "www.openstreetmap.org"
      ]
  }
}
```

Also adding to config.xml: 

```
<?xml version='1.0' encoding='utf-8'?>
<widget version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
  <access origin="*" />
  <allow-navigation href="*" />
</widget>
```

**Comment by dotNetkow** [comment]
I suggest debugging the app on iOS with a device connected to your computer. See [this guide](https://ionicframework.com/docs/troubleshooting/debugging#ios-and-safari). Launch the app, open Safari's devtools and inspect what the error is.

**Reply by syshex** [reply]
Unfortunately this is not possible with a physical Device.

I'll try it with the simulator, since that also show the same problem.

**Reply by aecheb** [reply]
Hi. Did you find the error? I have the same issue.

**Reply by nickguimond** [reply]
+1

**Reply by rodneyooo** [reply]
I have same issue.

**Comment by venn24** [comment]
You should proxy link from youtube to server and allow
